pkgs/by-name/gr/grimblast/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, makeWrapper
+, scdoc
+, coreutils
+, grim
+, jq
+, libnotify
+, slurp
+, wl-clipboard
+, hyprland ? null
+}:
+
+stdenvNoCC.mkDerivation {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

So that we can use it on `sourceRoot`.